Framework/Kernel/inc/MantidKernel/V3D.h

```diff
@@ -28,10 +28,11 @@ Class for 3D vectors.
 @author Laurent C Chapon, ISIS, RAL
 @date 09/10/2007
 */
-class MANTID_KERNEL_DLL V3D {
+class MANTID_KERNEL_DLL V3D final {
```

Comment:
Nice improvements here. I recently inlined much of `V2D` and this gave a big performance boost.

Reply:
Much of `V3D` was already inlined, so I don't expect much difference in performance. This was mostly a small modernization operation, anyway.